src/savefile/savefile.c

```diff
@@ -125,7 +118,7 @@ void savefileLoad() {
     gSramHandle.domain = PI_DOMAIN2;
     gSramHandle.speed = 0;
 
-    osCreateMesgQueue(&timerQueue, &timerQueueBuf, 1);
+    timeInit();
```

Comment:
Sleeping is only used here, but the time system as a whole is unrelated to the savefile. Can you move initialization to `main()`?